Wrapping/Generators/Python/itkExtras.py

```diff
@@ -147,19 +148,13 @@ def force_load():
         getattr(itk, k)
 
 
-import sys
-
-
-def echo(object, f=sys.stderr):
-    """Print an object is f
+def echo(obj, f=system_error_stream):
```

Comment:
renaming `object` into `obj` seems counter to ITK name guidelines. Is that some kind of Python practice?

Reply:
It is not a python convention.  I would be hesitant to apply C++ naming rules to python code like this.

If I recall correctly it was a way to avoid a "Shadow Variable" warning with the calling environment. I'd need to dig deeper to confirm that.